src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -182,6 +205,112 @@ public void checkTrialAccounts() {
         mailService.sendTrialAccountExpiresMail(DAYS_TO_CHECK, userDTOS);
     }
 
+    @GetMapping(path = "/check-exposed-tokens")
+    public void exposedTokensInfo() throws IOException, InterruptedException {
```

Comment:
Let's keep the method name same as the api url.
```suggestion
    public void checkExposedTokens() throws IOException, InterruptedException {
```